model/registry/k8s.yaml

```diff
@@ -104,6 +104,13 @@ groups:
         brief: >
           Last terminated reason of the Container.
         examples: ["Evicted", "Error"]
+      - id: container.status.waiting_reason
```

Comment:
Similar to `container.status.last_terminated_reason` should the name of this attribute convey a sense of time, like `container.status.latest_waiting_reason` or `container.status.current_waiting_reason`?